addons/overpressure/CfgEventHandlers.hpp

```diff
@@ -14,7 +14,7 @@ class Extended_PostInit_EventHandlers {
 class Extended_FiredBIS_EventHandlers {
     class CAManBase {
         class ADDON {
-            firedBIS = QUOTE(if (local (_this select 0)) then {_this call FUNC(firedEHBB);};);
+            clientFiredBIS = QUOTE(if (local (_this select 0)) then {_this call FUNC(firedEHBB);};);
```

Comment:
We already only ran on local and isPlayer which I just switched to == ace_player.  I think the idea was that AI have no idea how backblast works, so they would not know how to use it.

The firer will still trigger the targetEvent to everyone, so player shots will still effect AI. So skipping dedi on the event is just a very small performance savings.